ssp

```diff
@@ -2045,16 +2045,18 @@ sub print_ea3_php_configuration {
         }
 
         if ( $EA3_PHP4HANDLER && $EA3_PHP4HANDLER ne 'none' ) {
-            my @php_v = split /\n/, timed_run( 0, '/usr/local/php4/bin/php', '-v' );
-            if ( @php_v && $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ ) {
-                $EA3_PHP4VERSION = $1;
-            }
-            else {
-                $EA3_PHP4VERSION = '(version unknown)';
-            }
+            if ( -x '/usr/local/php4/bin/php' ) {
+                my @php_v = split /\n/, timed_run( 0, '/usr/local/php4/bin/php', '-v' );
+                if ( @php_v && $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ ) {
+                    $EA3_PHP4VERSION = $1;
+                }
+                else {
+                    $EA3_PHP4VERSION = '(version unknown)';
```

Comment:
Let's open up this phrase to be more specific.
```
$EA3_PHP4VERSION = '(ea3 php version unknown)';
```

We want to be sure that at a glance that communication is as specific as possible.

Reply:
this has been done per https://github.com/cPWilliamL/SSP/commit/f4aacec2bc7ca3a181878baeccb4462d9e13ab53

perlcritic: ✅
perltidy: ✅